cli/tests/environment-install.bats

```diff
@@ -167,3 +167,52 @@ teardown() {
 @test "i6: install on a pushed environment stages locally" {
   skip "remote environments handled in another phase"
 }
+
+@test "'flox install' installs by path" {
+  run "$FLOX_BIN" init;
+  assert_success;
+  run "$FLOX_BIN" install hello;
+  assert_success;
+  manifest=$(cat "$PROJECT_DIR/.flox/env/manifest.toml");
+  # This also checks that it correctly infers the install ID
+  assert_regex "$manifest" 'hello.path = "hello"';
+}
+
+@test "'flox install' infers install ID" {
+  run "$FLOX_BIN" init;
+  assert_success;
+  run "$FLOX_BIN" install rubyPackages_3_2.rails;
+  assert_success;
+  manifest=$(cat "$PROJECT_DIR/.flox/env/manifest.toml");
+  # This also checks that it correctly infers the install ID
+  assert_regex "$manifest" 'rails.path = "rubyPackages_3_2\.rails"';
```

Comment:
Maybe use assert_line here (unless assert_regex is already some wrapper?)
Also the "rails.path"  . Is not escaped

Reply:
"rails.path" doesn't need to be escaped

Reply:
wouldnt this also match `railsxpath`? or do i misinterpret the second arg as a regex?

Reply:
Ah, I thought you were referring to being escaped in the TOML since you wrote `"rails.path"`. Yes, the `.` needs to be escaped.